week-1/2-mandatory/4-tax.js

```diff
@@ -1,29 +1,18 @@
-/*
-  SALES TAX
-  =========
-  A business requires a program that calculates how much sales tax to charge
-  Sales tax is 20% of the price of the product
-*/
-
-function calculateSalesTax() {}
-
-/*
-  CURRENCY FORMATTING
-  ===================
-  The business has informed you that prices must have 2 decimal places
-  They must also start with the currency symbol
-  Write a function that transforms numbers into the format £0.00
-
-  Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
-*/
-
-function formatCurrency() {}
+function sTax(Price) {
```

Comment:
Use lowercase (camelCase) for javascript variables, functions and params. 
We only use uppercase for Classes and Components. Also - always give functions readable names that explain what the function does. This is not always easy but try to take the time as it has huge benefits for the readability of your code.